src/lib/index.ts

```diff
@@ -119,3 +121,20 @@ export async function assetsWidget({
     })
     return new module.AssetView({ asset, permissions })
 }
+
+export async function explorerViewModule() {
+    const module: typeof ExplorerViewModule =
+        await setup.installAuxiliaryModule({
+            name: 'explorer',
+            cdnClient: webpmClient,
+        })
+    return module
+}
+export async function explorerState() {
```

Comment:
For now, let's only expose 'explorerViewModule', we'll see with the refactoring how to expose a standalone 'ExplorerWidget'.

Reply:
also rename it 'explorerModule' (not explorerViewModule).